pkg/dockerregistry/server/auth_test.go

```diff
@@ -62,25 +82,26 @@ func TestVerifyImageStreamAccess(t *testing.T) {
 					Reason:    "authorized!",
 				}),
 			},
-			expectedError: nil,
 		},
 	}
 	for _, test := range tests {
 		ctx := context.Background()
 		server, _ := simulateOpenShiftMaster([]response{test.openshiftResponse})
+		defer server.Close()
```

Comment:
using defer in a loop will stack up servers until the function returns. nest in an anonymous function if you want to use defer